@ui/ui-kit/Textareas/Textarea.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ChangeEvent, FocusEvent } from 'react'
+import classNames from 'classnames'
+import styles from './TextArea.module.scss'
+import { TEXTAEREA_COLORS } from './constants'
+
+type TextAreaProps = {
+  defaultValue?: string
+  isActive?: boolean
+  hasError?: boolean
+  isHovered?: boolean
+  isFocused?: boolean
+  isDisabled?: boolean
+  errorMessage?: string
+  placeholderText?: string
+  color?: (typeof TEXTAEREA_COLORS)[keyof typeof TEXTAEREA_COLORS]
+  onFocus?: (event: FocusEvent<HTMLTextAreaElement>) => void
+  onChange?: (event: ChangeEvent<HTMLTextAreaElement>) => void
+  onBlur?: (event: FocusEvent<HTMLTextAreaElement>) => void
+}
+
+const TextArea: React.FC<TextAreaProps> = ({
+  defaultValue,
+  isActive = false,
+  hasError = false,
+  isHovered = false,
+  isFocused = false,
+  isDisabled = false,
```

Comment:
- isDisabled -> disabled 
- we dont need to have isHovered, isFocused etc... in props -  it`s pseudoclasses , use :hover :focus is SCSS

Reply:
tooooo much props for such little component , think about how difficult is using such Component